contrib/cmd/recvtty/recvtty.go

```diff
@@ -61,28 +61,35 @@ terminals:
 )
 
 func bail(err error) {
-	fmt.Fprintf(os.Stderr, "[recvtty] fatal error: %v\n", err)
+	_, _ = fmt.Fprintf(os.Stderr, "[recvtty] fatal error: %v\n", err)
 	os.Exit(1)
 }
 
-func handleSingle(path string, noStdin bool) error {
+func handleSingle(path string, noStdin bool) (err error) {
```

Comment:
I'd suggest calling it `retErr` or so, to distinguish between errors.

Reply:
`Err` is what I usually use.